packages/discovery-provider/.vscode/settings.json

```diff
@@ -15,7 +15,7 @@
   "python.testing.unittestEnabled": false,
   "[python]": {
     "editor.codeActionsOnSave": {
-      "source.organizeImports": true
+      "source.organizeImports": "explicit"
```

Comment:
thank you kindly - maybe call this out in client channel. or maybe smarter to put this stuff into its own PR? appreciate you doing this tho!

Reply:
Yeah I can remove it from this PR. It keeps making its way into my commits 😁